src/gwc/src/main/java/org/geoserver/gwc/GWC.java

```diff
@@ -820,6 +819,14 @@ public final ConveyorTile dispatch(
         return tileResp;
     }
 
+    private String getPrefixedName(String layerName) {
+        PublishedInfo info = catalog.getLayerByName(layerName);
+        if (info == null) info = catalog.getLayerGroupByName(layerName);
+        if (info != null) return info.prefixedName();
+        if (log.isLoggable(Level.SEVERE)) log.severe("Unable to find and prefix " + layerName);
```

Comment:
Severe? That would appear even in production logging, it's too much, consider this would be a client error (if the client uses the names reported in the capabilities documents, there should be no issue). Level should be INFO at most, if not FINE.

Also the mssage, "Unable to find and prefix " --> "Unable to fid a prefix for: "

Reply:
understood, will change

Reply:
this is changed